src/Controls/src/Core/ItemsView.cs

```diff
@@ -1,10 +1,11 @@
 using System.Collections;
+using System.Collections.Generic;
 using System.ComponentModel;
 using Microsoft.Maui.Controls.Internals;
 
 namespace Microsoft.Maui.Controls
 {
-	public abstract class ItemsView<TVisual> : View, ITemplatedItemsView<TVisual> where TVisual : BindableObject
+	public abstract class ItemsView<TVisual> : View, IVisualTreeHelper, ITemplatedItemsView<TVisual> where TVisual : BindableObject
```

Comment:
`View` inherits from `Element` so will the second implementation cause issues? For example, I might cast to `Element` and get something different?

Reply:
I've updated and removed it from the current implementation. As written, it would have only returned the actual ListView in a new list, rather than any of its children. While the base implementation won't work either, it's something that can be updated later.